app/javascript/components/ToggleSwitch/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const ToggleSwitch = ({ name, value, onChange, optionA, optionB }) => (
```

Comment:
Shamelessly ripped from https://www.sitepoint.com/react-toggle-switch-reusable-component/ with a few modifications. Styles came from here directly as well.

Reply:
I was just complimenting you to James the other day about how nice this looked hahaha